nlbq/core.py

```diff
@@ -46,7 +59,7 @@ def get_prompt_template(self) -> str:
         uncommented_lines = [line for line in lines if not line.strip().startswith("#")]
         return "\n".join(uncommented_lines).strip()
 
-    async def text_to_bq(self, question: str) -> str:
+    async def text_to_bq(self, question: str) -> Tuple:
```

Comment:
@tomdyson `main` now has pre-commit (and is a `lint` dependency)
```sh
python -m pip install ".[lint]"
pre-commit install
```

should sort you out. Also our min Python version is 3.11, so `-> tuple` should suffice

Reply:
Thanks @zerolab!